src/plugins/plan/pluginSpec.js

```diff
@@ -155,11 +156,19 @@ describe('the plugin', function () {
         });
 
         it('displays the activities and their labels', () => {
-            const rectEls = element.querySelectorAll('.c-plan__contents rect');
-            expect(rectEls.length).toEqual(2);
-            const textEls = element.querySelectorAll('.c-plan__contents text');
-            expect(textEls.length).toEqual(3);
+            const bounds = {
+                start: 1597160002854,
+                end: 1597181232854
+            };
+
+            openmct.time.bounds(bounds);
+
+            Vue.nextTick(() => {
```

Comment:
There's no call to `done()` and `Vue.nextTick()` is asynchronous. This is causing test state leakage right now.